ccp/src/main/java/com/togitech/ccp/component/TogiCountryCodePicker.kt

```diff
@@ -157,7 +157,13 @@ fun TogiCountryCodePicker(
             .focusable()
             .autofill(
                 autofillTypes = listOf(AutofillType.PhoneNumberNational),
-                onFill = { phoneNumber = it },
+                onFill = {
+                    phoneNumber = phoneNumberTransformation.preFilter(it)
```

Comment:
To address detekt complaint about multi line lambda https://github.com/jump-sdk/jetpack_compose_country_code_picker_emoji/actions/runs/6436523298/job/17528474579?pr=36
```suggestion
                onFill = { filledPhoneNumber ->
                    phoneNumber = phoneNumberTransformation.preFilter(filledPhoneNumber)
```